test_pull_requests

```diff
@@ -1184,8 +1288,55 @@ popd
         # comment to reflect the new state
         if submit_test_job
           delete_comment_with_prefix(id, base_repo, ACTION_PREFIX, comments)
-          extended_tests = get_extended_tests(req, comments, branch, settings)
-          submit_tests(repo_to_pull_request, base_repo, branch, comment_id, extended_tests, settings)
+          preceding_test_settings = Properties['settings'][settings['preceding_test_settings']]
+          test_build_url = ''
+          quick_merge_comments = []
+          can_skip_merge_tests =
+            if !preceding_test_settings.nil?
+              p_test_prefix = preceding_test_settings['test_prefix']
+              coreq_tests_synced_and_passed = repo_to_pull_request.all? do |repo, pull_request|
+                preceding_test_comment = get_comment_with_prefix(pull_request['number'], repo, p_test_prefix, (repo == base_repo ? comments : nil))
+                if preceding_test_comment.nil?
+                  false
+                else
+                  p_comment, p_test_build_url, p_base_commit, p_extended_tests = extract_bot_comment_metadata(preceding_test_comment['body'], preceding_test_settings)
+                  p_base_commit = base_commit_tag_to_sha(p_base_commit)
+                  can_skip_one = (p_comment =~ /SUCCESS/ &&
```

Comment:
How do you mean? FWIW, this string will contain just the bit of the bot comment between the comment prefix and the base URL, so maybe this is precise enough?